homework6/6.2/6.2/Game.cs

```diff
@@ -0,0 +1,114 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Task6_2;
+
+public class Game
+{
+    public List<List<char>> walls;
+    public (int left, int top) Position { get; set; } = (-1, -1);
+
+    public void LoadMap(string path)
+    {
+        using (StreamReader reader = new StreamReader(path))
+        {
+            int numberOfString = Console.CursorTop;
+            while (!reader.EndOfStream)
+            {
+                string inputString = reader.ReadLine();
+                walls.Add(new List<char>());
+                for (int i = Console.CursorLeft; i < inputString.Length;i++)
```

Comment:
```suggestion
                for (int i = Console.CursorLeft; i < inputString.Length; i++)
```
И я без идей, зачем тут Console.CursorLeft :)